processor/transformprocessor/README.md

```diff
@@ -347,6 +348,49 @@ Examples:
 
 - `copy_metric(desc="new desc") where description == "old desc"`
 
+### aggregate_on_attributes
```

Comment:
I think the docs need to explain a little more about what it means to aggregate a metric across attributes.  Or link to a doc that does explain the concept.

Reply:
adapted, please check if it's enough